flink-connector-hbase-2.2/src/test/java/org/apache/flink/connector/hbase2/source/HBaseRowDataAsyncLookupFunctionTest.java

```diff
@@ -74,8 +72,8 @@ public void testEval() throws Exception {
         assertTrue(result.size() < rowkeys.length);
         latch.await();
         lookupFunction.close();
-        List<String> sortResult =
-                Lists.newArrayList(result).stream().sorted().collect(Collectors.toList());
+        List<String> sortResult = new ArrayList<>(result);
+        Collections.sort(sortResult);
```

Comment:
The `result` is not used at all after line 72, so IMO simply calling `Collections.sort(result)` and assert `result` against `expected` would be enough here I think.